src/Toolkit/Collection.php

```diff
@@ -870,6 +870,25 @@ public function query(array $arguments = [])
         return $result;
     }
 
+    /**
+     * Returns a new collection consisting of random elements,
+     * from the original collection, shuffled or ordered
+     *
+     * @param int $count
+     * @param bool $shuffle
+     * @return $this|static
```

Comment:
Can this ever be `$this`? If I read it correctly, the object is cloned no matter what. In this case:

```suggestion
     * @return static
```

Reply:
Tbh I copied this from a random other Method 😅